api/controllers/install/controller_test.go

```diff
@@ -190,65 +171,6 @@ func TestConfigureInstallation(t *testing.T) {
 	}
 }
 
-// TestIntegrationComputeCIDRs tests the CIDR computation with real networking utility
-func TestIntegrationComputeCIDRs(t *testing.T) {
```

Comment:
this comment is not correct. it is redundant and tested already in the utils package